views/form.pug

```diff
@@ -0,0 +1,9 @@
+extends index.pug
+
+block content
+  form(action='/submit-form' method=method)
```

Comment:
So here, the form activates the `/submit-form` endpoint using whatever method is passed up because we have either served this view from the `/form-get` or the `/form-post` endpoint?